pkg/apis/kops/validation.go

```diff
@@ -293,6 +293,18 @@ func (c *Cluster) Validate(strict bool) error {
 	}
 
 	// Topology support
+	if c.Spec.Topology == nil {
```

Comment:
Is there a single place where we populate defaults for all of our entry points? EG: `kops create cluster` `kops edit cluster` etc